draft-ietf-tls-tls13.md

```diff
@@ -3108,24 +2960,14 @@ for each scenario:
 
 | Mode | Handshake Context | Base Key |
 |------|-------------------|----------|
-| 0-RTT | ClientHello + ServerConfiguration + Server Certificate + CertificateRequest (where ServerConfiguration, etc. are from the previous handshake) | xSS |
-| 1-RTT (Server) | ClientHello ... ServerConfiguration | master secret |
+| 0-RTT | ClientHello | xSS |
+| 1-RTT (Server) | ClientHello ... CertificateVerify | master secret |
```

Comment:
Assuming that's [Key Schedule](https://tlswg.github.io/tls13-spec/#key-schedule), I have no idea what you are talking about.

My point was that the CertificateVerify and Finished already already Certificate, and Finished includes CertificateVerify.  They can't depend on themselves.